system-test/fixtures/sample/src/allowExpressionsJs.js

```diff
@@ -11,6 +11,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// eslint-disable-next-line node/no-missing-require
 require('@google-cloud/debug-agent').start({
```

Comment:
Maybe we should add `system-test/fixtures` to the `.eslintignore`?

Reply:
(here and below)

Reply:
added to `. eslintignore` and removed the disabling lines
Thanks